pages/common/unrar.md

```diff
@@ -6,6 +6,10 @@
 
 `unrar x {{compressed.rar}}`
 
+- Extract files to existing path with original directory structure:
```

Comment:
I'm unsure I understood this description. It is kind of confusing to me. Would you mind to explain?
As far as I understand this command extracts the content of a compressed file into a directory. What do you mean by "with original directory structure"?

Reply:
Oh, I think I get it now. 

Does this mean that the file would be extracted and a new directory would be created?
So the final result would be something like this:
`c:/exctracted_path/extracted_files`
instead of this:
`c:/exctracted_files`

Is my understanding correct?